app/util/AWS.scala

```diff
@@ -37,9 +47,17 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val kinesisReindexStreamName = config.getString("aws.kinesis.reindexStreamName").get
 
-  lazy val kinesisClient = region.createClient(
+  lazy val stage = config.getString("stage").getOrElse("DEV")
+  lazy val readFromComposerAccount = config.getString("readFromComposer").getOrElse("false")
```

Comment:
I think you can do `config.getBoolean` and avoid the string conversion later on.